envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/debug/SnapshotDebugInfo.kt

```diff
@@ -28,8 +28,8 @@ data class Versions(
 )
 
 data class Snapshot(
-    val clusters: Map<String, Cluster>,
-    val endpoints: Map<String, ClusterLoadAssignment>,
+    val clusters: Map<String, Cluster> = emptyMap(),
```

Comment:
Are you sure adding default value does not break anything?

Reply:
There should be no problem. This is debugging endpoint and we always change `Snapshot` value with one from snapshot.